types/react-clock/react-clock-tests.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import Clock, { ClockProps } from 'react-clock';
+
+function MyApp() {
+    const [value] = React.useState<ClockProps['value']>(new Date());
```

Comment:
what about:
```suggestion
import Clock, { ClockValue } from 'react-clock';

function MyApp() {
    const [value] = React.useState<ClockValue>(new Date());
```
to avoid that strange index accessor. It' would be `ClockValue`-type anyway, would it?
I could be wrong, not using hooks much :)

Reply:
Yeah, that's better thanks